src/ape/managers/project/manager.py

```diff
@@ -409,6 +422,8 @@ def extensions_with_missing_compilers(self, extensions: Optional[List[str]]) ->
             List[str]: A list of file extensions found in the ``contracts/`` directory
             that do not have associated compilers installed.
         """
+        extensions = extensions or []
+
```

Comment:
```suggestion
```

You can remove this line because you return `extensions_found` on line 447 that is initially set as an empty list on line 427

Reply:
I think this is to make it so that the argument can be None if I understand correctly?

Reply:
@helloibis thanks that helps alot :)